app/models/planting.rb

```diff
@@ -8,6 +8,8 @@ class Planting < ActiveRecord::Base
   belongs_to :crop, counter_cache: true
   has_many :harvests, -> { order(harvested_at: :desc) }, dependent: :destroy
 
+  before_save :calc_and_set_days_before_maturity
```

Comment:
Yeah; that sort of thing. 

It's probably not a big deal right now, but if we did something like "... and when a planting is probably matured; issue an email prompt to harvest it"; for example; it wouldn't make as much sense in a before_save.